frontend/src/components/uam/components/users-table/users-table.tsx

```diff
@@ -32,8 +33,18 @@ const UsersTable: FC = () => {
     );
   }, [page]);
 
-  const columns: Column<UsersGetResponseDto>[] = getUsersColumns();
-  const rows: UsersGetResponseDto[] = getUsersRows(users.items);
+  const handleUserDelete = (userId: string): void => {
+    dispatch(uamActions.deleteUser({ id: userId }));
+    dispatch(
+      uamActions.getUsers({
+        page,
+        count: PaginationDefaultValue.DEFAULT_COUNT,
+      }),
+    );
+  };
```

Comment:
First of all, second action should wait untill first one will be finished. But, it would be better to filter current users with id, that deleteUser returns without making a getUsers request